testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/ejb/ssl/SSLEJBRemoteRemotingClientTestCase.java

```diff
@@ -0,0 +1,432 @@
+        addKeyStoreOp.get(PATH).set(path);
+        ModelNode credentialReference = new ModelNode();
+        credentialReference.get("clear-text").set(keyPass);
+        addKeyStoreOp.get("credential-reference").set(credentialReference);
+        addKeyStoreOp.get("type").set("JKS");
+        return addKeyStoreOp;
+    }
+
+    private static PathAddress getKeyManagerAddress(String keyManagerName) {
+        return PathAddress.pathAddress()
+                .append(SUBSYSTEM, "elytron")
+                .append("key-manager", keyManagerName);
+    }
+
+    private static ModelNode getAddKeyManagerOp(String keyManagerName, String keyStoreName, String keyPass) {
+        ModelNode addKeyStoreOp = Util.createAddOperation(getKeyManagerAddress(keyManagerName));
+        addKeyStoreOp.get("key-store").set(keyStoreName);
+        ModelNode credentialReference = new ModelNode();
+        credentialReference.get("clear-text").set(keyPass);
+        addKeyStoreOp.get("credential-reference").set(credentialReference);
+        return addKeyStoreOp;
+    }
+
+    private static PathAddress getTrustManagerAddress(String trustManagerName) {
+        return PathAddress.pathAddress()
+                .append(SUBSYSTEM, "elytron")
+                .append("trust-manager", trustManagerName);
+    }
+
+    private static ModelNode getAddTrustManagerOp(String trustManagerName, String keyStoreName) {
+        ModelNode addKeyStoreOp = Util.createAddOperation(getTrustManagerAddress(trustManagerName));
+        addKeyStoreOp.get("key-store").set(keyStoreName);
+        return addKeyStoreOp;
+    }
+
+    private static PathAddress getServerSSLContextAddress(String serverSSLContextName) {
+        return PathAddress.pathAddress()
+                .append(SUBSYSTEM, "elytron")
+                .append("server-ssl-context", serverSSLContextName);
+    }
+
+    private static ModelNode getAddServerSSLContextOp(String serverSSLContextName, String keyManagerName, String trustManagerName) {
+        ModelNode addServerSSLContextOp = Util.createAddOperation(getServerSSLContextAddress(serverSSLContextName));
+        addServerSSLContextOp.get("trust-manager").set(trustManagerName);
+        addServerSSLContextOp.get("need-client-auth").set(true);
+        addServerSSLContextOp.get("key-manager").set(keyManagerName);
+        return addServerSSLContextOp;
+    }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
+        System.setProperty("jboss.ejb.client.properties.skip.classloader.scan", "false");
+    }
+
+    private Properties getEjbClientContextProperties(String protocol, int port) throws IOException {
+        Properties props = new Properties();
+        props.put(Context.INITIAL_CONTEXT_FACTORY,
+                "org.wildfly.naming.client.WildFlyInitialContextFactory");
+        props.put(Context.PROVIDER_URL, String.format("%s://%s:%d", protocol, "127.0.0.1", port));
```

Comment:
"127.0.0.1" this could be relevant for the CI, try using `TestSuiteEnvironment.getServerAddress()` instead

Reply:
yeah, missed that and it was the cause of ipv6 failures